### ISSUE TITLE
Custom `letterSpacing` is ugly

### ISSUE THREAD

**Issue by jawj** [bluesky-social/social-app#5004]
**Describe the bug**

Letters are unusually widely spaced because CSS style `letter-spacing: 0.25` is applied widely in the stylesheet. This is strange and (IMHO) very ugly: to me, it makes _everything_ feel slightly off.

**To reproduce**

Use the app or website. Observe letter spacing.

**Expected behavior**

Letter spacing should not be altered. If letter spacing is altered, there should be a user preference to apply normal letter spacing instead.

**Screenshots**

I attach a screenshot of a post as it looks now, and the same post with the custom `letter-spacing` removed (by disabling the CSS rule in all relevant locations).

**Details**

 - Platform: Chrome/Mac and iOS app
 - Platform version: latest
 - App version: latest

**Additional context**

![current-letter-spacing](https://github.com/user-attachments/assets/327fda8a-fb88-4862-b7fc-bdf57b65c690)
![normal-letter-spacing](https://github.com/user-attachments/assets/fcaef051-94db-4f66-9e85-f3fee8199a79)

**Comment by mozzius** [comment]
Is this still an issue? iirc this was fixed maybe?

**Comment by jawj** [comment]
Yep, this was fixed some time ago. Thanks to whoever made that happen!
